clang/test.cpp

```
@@ -0,0 +1,88 @@
+    return n * 100;
+}
+
+void compareFloats(float a, float b) {
+    if (a == b) {
+        cout << "equal" << endl;
+    }
+}
+
+int main(int argc, char** argv) {
+    int uninitialized;
+    int result = Add(uninitialized, 7);
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)